layouts/shortcodes/sign-language-section.html

```diff
@@ -0,0 +1,19 @@
+{{/*
+
+  Usage: {{% sign-language-section VIDEO_ID %}}
+
+  */ -}}
+
+{{ $youtube := "" -}}
+{{ $videoID := .Get 0 -}}
+{{ if $videoID -}}
+  {{ $youtube = printf "{{< youtube %s >}}" $videoID }}
+{{ else -}}
+  {{ warnf "%s: Shortcode 'sign-section' requires a video ID as argument" .Page.Path -}}
+{{ end -}}
+
+## {{ i18n "sign_language_header" }}
+
+{{ trim " " (i18n "sign_language_info") }}
```

Comment:
This matter may require discussion with the sign language team (and WG) as well. Originally, there was a proposal to explicitly display the existing text, and consensus was reached on that.

At this point, I think it's best to keep it as is and consider it as a future improvement. However, I'll go through further discussions in #3406.

From a shortcode layout perspective, I’d like to ask for your opinion: If we keep the header and text as they are, would the following change be appropriate?

```diff
-{{ trim " " (i18n "sign_language_info") }}
+{{ i18n "sign_language_info" }} 
```

Reply:
Agreed, and fixed.